sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsRequestOptions.cs

```diff
@@ -46,5 +46,14 @@ internal TextAnalyticsRequestOptions(bool includeStatistics, string modelVersion
         /// <see cref="StringIndexType.Utf16CodeUnit"/>.
         /// </summary>
         public StringIndexType StringIndexType { get; set; } = StringIndexType.Utf16CodeUnit;
+
+        /// <summary>
+        /// The default value is 'false', '.
+        /// By default (except for methods like 'StartAnalyzeHealthcareEntities' and 'RecognizePiiEntities), Text Analytics
+        /// logs your input text for 48 hours, solely to allow for troubleshooting issues.
+        /// The default value of this property is 'false', except for methods like 'StartAnalyzeHealthcareEntities' and 'RecognizePiiEntities.
```

Comment:
I think you're adding duplicate information by accident.
```
The default value is 'false', '.
By default (except for methods like 'StartAnalyzeHealthcareEntities' and 'RecognizePiiEntities), Text Analytics
logs your input text for 48 hours, solely to allow for troubleshooting issues.
```
seems to be a superset of:
```
The default value of this property is 'false', except for methods like 'StartAnalyzeHealthcareEntities' and 'RecognizePiiEntities.
```

Reply:
Also, `'RecognizePiiEntities` is missing its closing quote ' in both occurrences.

Reply:
I'm sorry :(